ods_ci/tests/Tests/100__deploy/130__operators/140__serverless_operator/140__serverless_operator.robot

```diff
@@ -0,0 +1,111 @@
+*** Settings ***
+Documentation    Test Cases to verify Serverless installation
+Library         Collections
+Library         SeleniumLibrary
+Library         OpenShiftLibrary
+Resource        ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Suite Setup      Suite Setup
+Suite Teardown   Suite Teardown
+
+
+*** Variables ***
+${RHOAI_APPNAME}  Red Hat OpenShift AI
+${RHOAI_OPERATOR_NAME}    Red Hat OpenShift AI
+${KSERVE_APPNAME}  Red Hat OpenShift Serverless
+${KSERVE_OPERATOR_NAME}    Red Hat OpenShift Serverless
+${KNATIVESERVING_NS}    knative-serving
+${RHOAI_OPERATOR_NS}    redhat-ods-operator
```

Comment:
Can't we use the `OPERATOR_NAMESPACE` property defined in `test-variables.yml`?

Reply:
Changed @jstourac